projects/website/src/sitemap.xml

```diff
@@ -300,6 +300,10 @@
         <loc>https://clarity.design/news</loc>
         <changefreq>daily</changefreq>
     </url>
+    <url>
```

Comment:
Thanks for this!
Need to check where we miss updating the sitemap during release.